config/teleop.yaml

```diff
@@ -53,3 +53,8 @@ teleop:
     dampen: 3
     pan: 4
     tilt: 5
+
+  echo_presets:
+    blah: ["/tf", "/tf_static", "/rosout"]
+    foo: ["/rosout", "/tf"]
+    bar: ["/tf_static", "/rosout", "/tf"]
```

Comment:
as someone who hasn't worked on the echo presets at all, why are they named like that

Reply:
These are just from testing. I don't know what presets/names we'd use yet